samanthas_telegram_bot/data_structures/constants.py

```diff
@@ -0,0 +1,43 @@
+import re
+from typing import Literal
+
+# Most of these could come from the backend to avoid duplicate changes, but that would mean
+# the bot phrases will have to be stored in the backend too (since these types are used to identify
+# the phrases).
+# TODO (not necessarily in MVP) maybe this can be changed anyway.
+#  We could check ID's of phrases at the start to make sure there's no mismatch.
+
+EMAIL_PATTERN = re.compile(r"^([\w\-.]+)@([\w\-.]+)\.([a-zA-Z]{2,5})$")
+
+LOW_LEVELS = ("A0", "A1")
+LEVELS_ELIGIBLE_FOR_ORAL_TEST = ("A2", "B1", "B2", "C1", "C2")
+ALL_LEVELS = LOW_LEVELS + LEVELS_ELIGIBLE_FOR_ORAL_TEST
+
+Locale = Literal["ua", "en", "ru"]
+LOCALES: tuple[Locale, ...] = ("ua", "en", "ru")
+
+NON_TEACHING_HELP_TYPES = (
```

Comment:
Don't you think, we need to make an API for this enums in backend? Like you get assessments or age ranges. Just to remove some duplication

Reply:
Yes, removing duplication would be great. For now, though, looks like TODO :)

Reply:
Let's make an issue in backend repo :)